ui/src/timeMachine/actions/queries.ts

```diff
@@ -194,13 +201,15 @@ export const executeQueries = () => async (dispatch, getState: GetState) => {
     dispatch(
       setQueryResults(RemoteDataState.Done, files, duration, null, statuses)
     )
-  } catch (e) {
-    if (e.name === 'CancellationError') {
+    return results
+  } catch (error) {
+    if (error.name === 'CancellationError' || error.name === 'AbortError') {
```

Comment:
👍

Reply:
This showed up cause of testing! There are situations where the request can be canceled before the other the `catch` in `runQuery` has a chance to catch it, so it would wind up here without properly being converted into a `CancellationError`